ere-guests/Cargo.toml

```diff
@@ -138,12 +138,22 @@ risc0-zkvm = { version = "3.0.3", default-features = false, features = [
     "unstable",
     "getrandom",
 ] }
+
 openvm = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.4.1", features = [
     "std",
 ] }
-openvm-keccak256 = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.4.1" }
-openvm-algebra-guest = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.4.1" }
-openvm-ecc-guest = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.4.1" }
+openvm-ecc-guest = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.4.1", default-features = false }
+openvm-sha2 = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.4.1", default-features = false }
+openvm-pairing = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.4.1", default-features = false, features = [
+    "bn254",
+    "bls12_381",
+] }
+openvm-k256 = { git = "https://github.com/openvm-org/openvm.git", package = "k256", tag = "v1.4.1" }
+openvm-p256 = { git = "https://github.com/openvm-org/openvm.git", package = "p256", tag = "v1.4.1" }
+openvm-kzg = { git = "https://github.com/axiom-crypto/openvm-kzg.git", branch = "openvm-main", default-features = false }
```

Comment:
This `branch =` reference is the same used in https://github.com/axiom-crypto/openvm-reth-benchmark/pull/523/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R153.

@jonathanpwang, is that branch kind of frozen? I think prob is better to also have a tag 1.4.1 or similar?